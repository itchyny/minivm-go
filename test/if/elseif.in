if false
  a = 10
  print a
elseif false
  b = 11
  print b
elseif true
  c = 12
  print c
else
  d = 13
  print d
end

e = 14
print e

if false
  a = 10
  print a
elseif true
  b = 11
  print b
end
