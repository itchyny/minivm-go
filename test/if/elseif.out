12
14
11
